packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,195 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+import { IconCross } from '@frontify/fondue-icons';
+import * as RadixDialog from '@radix-ui/react-dialog';
+import { forwardRef, type CSSProperties, type ForwardedRef, type ReactNode } from 'react';
+
+import {
+    dialogBodyStyles,
+    dialogContentStyles,
+    dialogFooterStyles,
+    dialogHeaderStyles,
+    dialogSideContentStyles,
+    dialogUnderlayStyles,
+} from './styles/DialogStyles';
+
+export type DialogRootProps = {
+    /**
+     * Disable Interaction with the rest of the page
+     * @default false
+     */
+    modal?: boolean;
+    /**
+     * The controlled `open` state of the Dialog
+     */
+    open?: boolean;
+    /**
+     * callback when the open state changes
+     */
+    onOpenChange?: (open: boolean) => void;
+    children?: ReactNode;
+};
+
+export type DialogContentProps = {
+    /**
+     * Add rounded corners to the Dialog
+     * @default true
+     */
+    rounded?: boolean;
+    /**
+     * Define the padding of the Dialog
+     * @default compact
+     */
+    padding?: 'compact' | 'comfortable' | 'spacious';
+    /**
+     * Override the `800px` maximum width of the Dialog
```

Comment:
Could you remove the `"Override the "XYZpx"` and put `Define the` for all?
e.g. `Define the maximum width of the dialog` (lowercased **d**ialog 😉 )

Reply:
changed